src/components/ContributeAs.js

```diff
@@ -51,28 +51,28 @@ const enhance = compose(
 
       return onChange(selected.value);
     },
-    onFieldChange: ({ onChange, setState }) => event => {
+    onFieldChange: ({ onChange, setState, state }) => event => {
       event.stopPropagation();
 
       const { target } = event;
       if (!target.validity.valid) {
-        target.reportValidity();
+        debounce(onChange, 100)(null);
```

Comment:
This has no effect. Debounce returns a new debounced function, each one has its own state. It does not store a global register of functions. You're suppose to instantiate it only one time, then use this instance each time.

See this example: http://jsfiddle.net/u1embar3/1/

Reply:
Ah ok. I'll update this.